scripts/update_golang_version.sh

```diff
@@ -9,16 +9,19 @@ fi
 
 golang_version="$1"
 
-current_version=$(grep -o -e 'golang:[.0-9]*' .circleci/config.yml | head -n1 | cut -d: -f2)
+current_version=$(grep -o -E -e '/golang:[.0-9]+' .circleci/config.yml | head -n1 | cut -d: -f2)
```

Comment:
Old regex matched the `install-golang:` lines in config.yml.

I think only the `/` prefix *or* the `-E` and `+` are required, not both. Let me know if I should simpify it.